cmd/kafka_consumer_group_exporter.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"context"
+	"net/http"
+	"os"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+
+	"github.com/kawamuray/prometheus-kafka-consumer-group-exporter/kafka"
+	kafkaprom "github.com/kawamuray/prometheus-kafka-consumer-group-exporter/prometheus"
+	"github.com/kawamuray/prometheus-kafka-consumer-group-exporter/sync"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/urfave/cli"
+)
+
+const consumerGroupCommandName = "kafka-consumer-groups.sh"
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "kafka_consumer_group_exporter"
+	app.Version = version
+	app.Usage = "[OPTIONS] BOOTSTRAP_SERVER"
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "consumer-group-command-path",
+			Usage: "Path to `kafka-consumer-groups.sh`.",
+			Value: consumerGroupCommandName,
+		},
+		cli.StringFlag{
+			Name:  "listen",
+			Usage: "Interface and port to listen on.",
+			Value: ":7979",
+		},
+		cli.DurationFlag{
+			Name:  "kafka-command-timeout",
+			Usage: "The maximum time the Kafka command is allowed to take before we kill it. We've seen it block forever in production at times (most likely during rebalances).",
+			Value: 5 * time.Minute,
+		},
+		cli.IntFlag{
+			Name:  "max-concurrent-group-queries",
+			Usage: "The maximum number of consumer groups that are queried concurrently.",
+			Value: 20,
```

Comment:
Very good point! Corrected in 4311a211a091aa4c66e29364773820c1b2593b44. Also added a code comment about the reasoning behind the default value just in case.